crates/eips/src/eip7702/auth_list.rs

```diff
@@ -160,6 +164,32 @@ impl Deref for SignedAuthorization {
     }
 }
 
+#[cfg(any(test, feature = "arbitrary"))]
+impl<'a> arbitrary::Arbitrary<'a> for SignedAuthorization {
+    fn arbitrary(u: &mut arbitrary::Unstructured<'a>) -> arbitrary::Result<Self> {
+        use alloy_primitives::Parity;
+
+        let inner = u.arbitrary::<Authorization>()?;
+        let parity = u.arbitrary::<Parity>()?;
+
+        // TODO: find an easy way to generate random signatures
```

Comment:
why not run it in a loop with random values until it works?

Reply:
tried this first, couldn't figure out what random values to use, random u256 just ran forever